doc/classes/DisplayServer.xml

```diff
@@ -1168,6 +1168,40 @@
 				[b]Note:[/b] This method is implemented on Android, iOS, macOS, Windows, and Linux (X11/Wayland).
 			</description>
 		</method>
+		<method name="simulate_keypress">
+			<return type="bool" />
+			<param index="0" name="keycode" type="int" enum="Key" />
+			<param index="1" name="modifiers" type="int" enum="KeyModifierMask" is_bitfield="true" />
+			<param index="2" name="pressed" type="bool" />
+			<description>
+				Simulates keypress event.
+				[b]Note:[/b] Simulated input events are ignored by login screen, and other security related OS UI elements.
+				[b]Note:[/b] On macOS, this method requires "Accessibility" permission.
+				[b]Note:[/b] This method is implemented on macOS, Windows, and Linux (X11).
+			</description>
+		</method>
+		<method name="simulate_mouse_click">
+			<return type="bool" />
+			<param index="0" name="button" type="int" enum="MouseButton" />
+			<param index="1" name="modifiers" type="int" enum="KeyModifierMask" is_bitfield="true" />
+			<param index="2" name="pressed" type="bool" />
+			<description>
+				Simulates mouse button press.
```

Comment:
```suggestion
				Simulates a mouse button or wheel event which can be handled by the operating system, not just Godot (as opposed to [method Input.parse_input_event]). The mouse button/wheel event is typically sent to the currently focused window, but some apps may have global shortcuts active. Note that sending a mouse click here can cause the currently focused application to change (depending on its current position).
```

Reply:
> The mouse button/wheel event is typically sent to the currently focused window, but some apps may have global shortcuts active.

I think it should be sent to the top window under the pointer (I'll check it later), mouse events usually do not care about focus.